KsApi/MockService.swift

```diff
@@ -736,11 +733,11 @@
       return SignalProducer(value: CategoryEnvelope(node: .template |> Category.lens.id .~ "\(query.head)"))
     }
 
-    internal func fetchGraphUser()
+    internal func fetchGraphUser(withStoredCards _: Bool)
```

Comment:
IS the underscore after the `withStoredCards` parameter needed here and the other altered methods above?

Reply:
We don't need it, just following the conventions above for things like `fetchProjectComments` and `fetchUpdateComments`, but okay removed in this call.

Reply:
ok nvm, `bin/format.sh .` does this.